SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -20,6 +21,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, EntityCreationRateLimit(0)
 	, OpsUpdateRate(30.0f)
 	, bUsingQBI(true)
+	, bUsingOffloading(true)
```

Comment:
Should we default this to false?

Reply:
I put it to true as we were gonna use it for scavengers but happy to make it false by default, shouldn't matter to us now that we have the engine change to capture when you have a mismatch in configs. Will update now:)

Reply:
Updated now PTAL:)